src/Workspaces/CoreTestUtilities/RenameHelpers.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Rename;
+using Roslyn.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.Utilities
+{
+    public static class RenameHelpers
+    {
+        public static async Task AssertRenameAnnotationsAsync(Solution originalSolution, Solution newSolution, IReadOnlyDictionary<string, string> symbolPairs, CancellationToken cancellationToken = default)
+        {
+            var allDocuments = newSolution.Projects.SelectMany(p => p.Documents);
+            var remainingSymbols = symbolPairs.ToHashSet();
+
+            foreach (var document in allDocuments)
```

Comment:
```suggestion
            foreach (var newDocuments in allDocuments)
```

So that way all locals are prefixed with 'original' or 'new' to preserve what little sanity I have left.